### PR TITLE
Update config

### PR DESCRIPTION
**Pull request by CameronRP**

### CODE REVIEW

File: cmd/leptond/config.go [hunk 1]
```diff
@@ -17,35 +17,34 @@
...
-	FrameOutput: "/var/run/lepton-frames",
+	SPISpeed    int64
+	PowerPin    string
+	FrameOutput string
 }
 
-func ParseConfigFile(filename string) (*Config, error) {
-	buf, err := ioutil.ReadFile(filename)
+func ParseConfigFile(configFolder string) (*Config, error) {
+	configRW, err := goconfig.New(configFolder)
```
Comment (mjs) [inline cmd/leptond/config.go:30]: what does "RW" mean?

File: cmd/leptond/config.go [hunk 2]
```diff
@@ -17,35 +17,34 @@
...
-	PowerPin:    "GPIO23",
-	FrameOutput: "/var/run/lepton-frames",
+	SPISpeed    int64
+	PowerPin    string
+	FrameOutput string
 }
 
-func ParseConfigFile(filename string) (*Config, error) {
-	buf, err := ioutil.ReadFile(filename)
+func ParseConfigFile(configFolder string) (*Config, error) {
```
Comment (mjs) [inline cmd/leptond/config.go]: Maybe rename this to just `ParseConfig`?

File: cmd/thermal-recorder/cptv_test.go [hunk 3]
```diff
@@ -21,20 +21,36 @@ import (
...
+	"runtime"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
-
+	config "github.com/TheCacophonyProject/go-config"
 	"github.com/TheCacophonyProject/thermal-recorder/motion"
 	"github.com/TheCacophonyProject/thermal-recorder/recorder"
+	"github.com/TheCacophonyProject/window"
+	"github.com/stretchr/testify/assert"
```
Comment (mjs) [inline cmd/thermal-recorder/cptv_test.go:30]: these two should be grouped with go-config above

**Review by mjs — APPROVED** [review]
LGTM

File: cmd/leptond/config.go (hunk 1 above)
Comment (CameronRP) [reply cmd/leptond/config.go:30]: ReadWrite.

**Pull request merged**